<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComponentSelectorWidget</class>
 <widget class="QWidget" name="ComponentSelectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>193</width>
    <height>66</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Component Selector</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QRadioButton" name="fileButton">
     <property name="text">
      <string>File:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="nameButton">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="newButton">
     <property name="text">
      <string>New:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="BrowseWidget" name="browseWidget" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="newWrapper" native="true">
     <layout class="QVBoxLayout" name="newLayout">
      <property name="margin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BrowseWidget</class>
   <extends>QWidget</extends>
   <header>BrowseWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>browseWidget</tabstop>
  <tabstop>nameEdit</tabstop>
  <tabstop>newWrapper</tabstop>
  <tabstop>fileButton</tabstop>
  <tabstop>nameButton</tabstop>
  <tabstop>newButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
